src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,67 @@
+.stopwatch {
+  &__container {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    background: rgb(2,0,30);
+    @include circle ($stopwatch-big-circle-diametr);
+    border: $color-grey 8px solid;
+    box-shadow: 0 0 5px black;
+    border-radius: 50%;
+  }
+
+  &__center-circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    @include circle ($stopwatch-small-circle-diametr);
+
+    background-color: $color-grey;
+    box-shadow: 0 0 10px black;
+  }
+
+  &__minutes-hand {
```

Comment:
A lot of code for hands can be reused. Pass differences as arguments to minimize the amount of repeated code.